src/main/java/handlers/HandlerGetDirectoryContents.java

```diff
@@ -18,20 +18,13 @@ public HttpResponse generateResponse() {
     try {
       String[] fileList = fileIO.getFiles(path);
       byte[] body = dirListHtml.buildHtmlPage(path, fileList);
-      rtnResponse.setBody(body);
-      addHeaders(rtnResponse, body);
+      rtnResponse.setBody(body, "text/html");
     }
     catch (IOException e) {
       e.printStackTrace();
-      rtnResponse.setBody("IOException");
+      rtnResponse.setBody("IOException", "text/plain");
```

Comment:
Hmmm ... I sort of feel like this should be a 500 now that I think about it. _Something_ went wrong on the server, right? If we get to this point we don't _expect_ an exception.

Do we do 500's?